extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -266,11 +280,15 @@ export class FileConfigPage extends ImportPage {
 
 	private async populateSchemaDropdown(): Promise<boolean> {
 		this.schemaLoader.loading = true;
+
 		let connectionUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
+		let connection = await azdata.connection.getConnection(connectionUri);
+		connection.databaseName = this.model.database;
+
 		let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(this.model.server.providerName, azdata.DataProviderType.QueryProvider);
 
 		const escapedQuotedDb = this.databaseDropdown.value ? `[${(<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]')}].` : '';
-		const query = `SELECT name FROM ${escapedQuotedDb}sys.schemas`;
+		let query = `SELECT name FROM ${escapedQuotedDb}sys.schemas`;
```

Comment:
Just leave this as const. It's good practice to default vars to const and if only if you need to modify them change to let.